encord/orm/label_row.py

```diff
@@ -307,4 +307,12 @@ def to_dict(self) -> dict:
         Returns:
             The dict equivalent of LabelRowMetadata.
         """
-        return asdict(self)
+
+        def transform(value: Any):
+            if isinstance(value, Enum):
+                return value.value
+            elif isinstance(value, datetime.datetime):
+                return value.isoformat()
```

Comment:
given that this PR is about consistency, are you sure that this is the format of the date that is coming in? I suspect it is going to be a different format.

Reply:
In the `from` method we use `fromisoformat` so I think it makes that this is the format coming out as well.